ClosedXML_Tests/Excel/Misc/XLWorkbookTests.cs

```diff
@@ -74,6 +74,85 @@ public void Cells3()
             Assert.AreEqual(1, cells.First().GetValue<Int32>());
         }
 
+        [Test]
+        public void GetCellFromFullAddress()
```

Comment:
Maybe just include a test for a sheet name with some spaces or other oddities?

Reply:
Done